staging/vhost-device-gpu/src/lib.rs

```diff
@@ -6,28 +6,45 @@ pub mod device;
 pub mod protocol;
 pub mod virtio_gpu;
 
+use serde::Deserialize;
 use std::path::PathBuf;
 
+#[derive(Clone, Copy, Debug, Deserialize)]
+pub enum GpuMode {
+    #[serde(rename = "virglrenderer", alias = "3d", alias = "3D")]
+    ModeVirglRenderer,
+    #[serde(rename = "gfxstream")]
+    ModeGfxstream,
+}
```

Comment:
Since you removed the manual parsing, we don't need to derive `Deserialize` here anymore and don't need the serde crate.  I am not sure if we need a separate enum with the same values, but I think makes sense to keep it, because we already have the `GpuConfig` / `GpuArgs` split.